controllers/cartController.js

```diff
@@ -0,0 +1,20 @@
+const { catchAsync } = require("../utils/error");
+const cartService = require("../services/cartsService");
+
+const getCartList = catchAsync(async (req, res) => {
+  const list = await cartService.getCartList(req.userId);
+
+  res.status(201).json({ data: list });
```

Comment:
```suggestion
  res.status(200).json({ data: list });
```
데이터 생성이나 수정이 아닌 Get이므로 200으로 바꿔주세요.